### PR TITLE
[W-10706803] Fix typos (part 2)

### PR DESCRIPTION
**Pull request by cheungaryk**
ref: [W-10706803](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000opXQBYA2/view)

Fixed a number of typos using VSCode and the code spell check extension. This is similar to #1810 .

Noteworthy changes:

- please pay closer attention to the workday attributes - they may or may not be typos (but I think they are)
- https://docs.mulesoft.com/veevavault-connector/1.3/veevavault-connector-examples#validating-a-spark-message now looks like this (organized a bit better):
  Screen Shot 2022-02-17 at 2 38 43 PM
- https://docs.mulesoft.com/servicenow-connector/0.3.5/#example-use-case had some bad blocks. They have been fixed and looks the same locally:
  Screen Shot 2022-02-17 at 2 44 19 PM
- https://docs.mulesoft.com/siebel-connector/0.3.5/#adding-the-oracle-siebel-business-objects-connector-to-a-flow had some image files that have typos. Both the references and the file names were corrected, and the image displayed correctly locally:
  Screen Shot 2022-02-17 at 2 54 17 PM
- https://docs.mulesoft.com/slack-connector/1.0/slack-connector-reference contains 2 identical sections of onNewFileTrigger and onNewMessageTrigger, 1 at the beginning and 1 at the end (cmd+f "New File Trigger" to see what I mean). I think the sections at the end are there by mistake and should be removed? In any case, I assigned them 2 different HTML id's for now because they were sharing the same one. Let me know what you want to do

### CODE REVIEW

File: salesforce-analytics-cloud/0.3.6/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -206,7 +206,7 @@ image::connectorconfig.jpg[]
 
 Create a data set in the Salesforce Analytics Cloud system, read the data from an input file and split it into batches, upload batches of data into the data set, and alert the system to start processing the data.
 
-Refer to documentation on xref:3.6@mule-runtime::poll-reference.adoc[B]xref:3.6@mule-runtime::batch-processing.adoc[atch Processor] and xref:6@studio::datamapper-user-guide-and-reference.adoc[Data Mapper] for in-depth information about these Mule elements.
+Refer to documentation on xref:3.6@mule-runtime::poll-reference.adoc[B]xref:3.6@mule-runtime::batch-processing.adoc[batch Processor] and xref:6@studio::datamapper-user-guide-and-reference.adoc[Data Mapper] for in-depth information about these Mule elements.
```
Comment (sathya0) [inline salesforce-analytics-cloud/0.3.6/modules/ROOT/pages/index.adoc]:
```suggestion
Refer to documentation on xref:3.6@mule-runtime::batch-processing.adoc[Batch Processor] and xref:6@studio::datamapper-user-guide-and-reference.adoc[Data Mapper] for in-depth information about these Mule elements.
```

File: salesforce-analytics-cloud/0.3.6/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -324,7 +324,7 @@ image::3config.jpg[]
...
```
Comment (sathya0) [inline salesforce-analytics-cloud/0.3.6/modules/ROOT/pages/index.adoc]:
```suggestion
Refer to documentation on xref:3.6@mule-runtime::batch-processing.adoc[Batch Processor] and xref:6@studio::datamapper-user-guide-and-reference.adoc[Data Mapper] for in-depth information about these Mule elements.
```

File: x12-edi/2.0/modules/ROOT/pages/x12-edi-schema-language-reference.adoc [hunk 3]
```diff
@@ -101,7 +101,7 @@ Group definitions are shown in expanded form, with key-value pairs on separate l
 * U for Unused, the item should not appear at all in data
 * I for Ignored, it can be present in data, (in the case of the HL segments it must be present) but doesn't show up for the user in metadata
 
-|count |mMximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
+|count |mMaximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```
Comment (sathya0) [inline x12-edi/2.0/modules/ROOT/pages/x12-edi-schema-language-reference.adoc]:
```suggestion
|count |Maximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```

File: x12-edi/2.1/modules/ROOT/pages/x12-edi-schema-language-reference.adoc [hunk 4]
```diff
@@ -102,7 +102,7 @@ Group definitions are shown in expanded form, with key-value pairs on separate l
 * U for Unused, the item should not appear at all in data
 * I for Ignored, it can be present in data, (in the case of the HL segments it must be present) but doesn't show up for the user in metadata
 
-|count |mMximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
+|count |mMaximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```
Comment (sathya0) [inline x12-edi/2.1/modules/ROOT/pages/x12-edi-schema-language-reference.adoc]:
```suggestion
|count |Maximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```

File: x12-edi/2.2/modules/ROOT/pages/x12-edi-schema-language-reference.adoc [hunk 5]
```diff
@@ -101,7 +101,7 @@ Group definitions are shown in expanded form, with key-value pairs on separate l
 * U for Unused, the item should not appear at all in data
 * I for Ignored, it can be present in data, (in the case of the HL segments it must be present) but doesn't show up for the user in metadata
 
-|count |mMximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
+|count |mMaximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```
Comment (sathya0) [inline x12-edi/2.2/modules/ROOT/pages/x12-edi-schema-language-reference.adoc]:
```suggestion
|count |Maximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```

File: x12-edi/2.3/modules/ROOT/pages/x12-edi-schema-language-reference.adoc [hunk 6]
```diff
@@ -101,7 +101,7 @@ Group definitions are shown in expanded form, with key-value pairs on separate l
 * U for Unused, the item should not appear at all in data
 * I for Ignored, it can be present in data, (in the case of the HL segments it must be present) but doesn't show up for the user in metadata
 
-|count |mMximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
+|count |mMaximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```
Comment (sathya0) [inline x12-edi/2.3/modules/ROOT/pages/x12-edi-schema-language-reference.adoc]:
```suggestion
|count |Maximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```

File: x12-edi/2.4/modules/ROOT/pages/x12-edi-schema-language-reference.adoc [hunk 7]
```diff
@@ -101,7 +101,7 @@ Group definitions are shown in expanded form, with key-value pairs on separate l
 * U for Unused, the item should not appear at all in data
 * I for Ignored, it may be present in data, (in the case of the HL segments it must be present) but doesn't show up for the user in metadata
 
-|count |mMximum repetition count value, which may be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
+|count |mMaximum repetition count value, which may be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```
Comment (sathya0) [inline x12-edi/2.4/modules/ROOT/pages/x12-edi-schema-language-reference.adoc]:
```suggestion
|count |Maximum repetition count value, which may be a number or the special value `>1` meaning any number of repeats (optional, count value of 1 is used if not specified)
```

**Comment by cheungaryk** [comment]
@sathya0 I did a fix on the duplicate headers on the Slack connector reference page: https://github.com/mulesoft/docs-connectors/blob/1b17d891ef538e3c60d84c85676533d782d85cae/slack/1.0/modules/ROOT/pages/slack-connector-reference.adoc. In short, the top-of-the-page references to OnNewFileTrigger and OnNewMessageTrigger have been removed, and the references towards the bottom of the page are kept.

Also applied all the suggested fixes. Please take another look, thank you!

**Review by sathya0 — APPROVED** [review]

**Pull request merged**